luxai2021/game/game.py

```diff
@@ -630,13 +673,14 @@ def spawn_city_tile(self, team, x, y, city_id=None):
         # now update the cities field accordingly
         adj_cells = self.map.get_adjacent_cells(cell)
 
-        city_ids_found = set()
+        city_ids_found = []
 
         adj_same_team_city_tiles = []
         for cell2 in adj_cells:
```

Comment:
nit (readability): replace `cell2` by `adjacent_cell`

Reply:
There is also `cell3`, and I actually prefer the readability as it is now, so I am not changing it at the moment.